apps/bfd-server/bfd-server-launcher/src/main/resources/logback.xml

```diff
@@ -56,6 +56,11 @@
 			</layout>
 		</encoder>
 	</appender>
+	
+	<!-- Enabling DEBUG level logging for hikari -->
```

Comment:
Do you know how much log spam this will generate? If it's a lot, please add a comment here explaining when it can be removed, e.g. "TODO: disable this once BLUEBUTTON-1234 is resolved"

Reply:
It looks like there will be quite a bit of spam. I've added this comment. Thanks.